src/test/java/com/mysql/cj/jdbc/NonRegisteringDriverTest.java

```diff
@@ -37,16 +37,45 @@
 import org.mockito.MockedStatic;
 import org.mockito.Mockito;
 
+import java.lang.reflect.Method;
 import java.sql.Connection;
 import java.sql.SQLException;
 import java.util.Properties;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
 import static org.junit.jupiter.api.Assertions.*;
 import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.*;
 
 @SuppressWarnings("unchecked")
 public class NonRegisteringDriverTest {
+
```

Comment:
should we maybe add a BeforeEach here that calls Driver.setAcceptAwsProtocolOnly(false) in case we add more tests later or change this one and don't want the static setting to carry over to other tests?